src/pem.rs

```diff
@@ -57,6 +57,7 @@ pub enum Item {
 
 /// Errors that may arise when parsing the contents of a PEM file
 #[derive(Debug, PartialEq)]
+#[non_exhaustive]
```

Comment:
Tiny nit: I think we usually put `#[non_exhaustive]` before the derives?

Reply:
Oops, reinstated.